.gitlab-ci.yml

```diff
@@ -78,6 +78,23 @@ default:
         - make -j $(nproc) check
         - make -j $(nproc) install
 
+.test-sched:
+    extends: .lc-variables
+    variables:
+        PYTHON: "/usr/bin/python3"
+        debug: t
+        FLUX_TESTS_LOGFILE: t
+    script:
+        - cd ${CI_DIRECTORY}
+        - export PKG_CONFIG_PATH=${CORE_INSTALL_PREFIX}/lib/pkgconfig:$(pkg-config --variable pc_path pkg-config)
+        - git clone https://github.com/flux-framework/flux-sched
+        - cd flux-sched
+        - module load gcc 
+        - ${CORE_INSTALL_PREFIX}/bin/flux start ./configure
```

Comment:
I meant to mention that you could probably drop the `start` here and just use `/path/to/flux ./configure`. This is because when an explicit path is used with `flux(1)` it will execute its argument with the same environment as would have been passed to an subcommand. 

Try that and see if it works if you'd like.

Reply:
Oh, that's handy, thanks! Fixed and pushed. 

I also noticed that the commit message was out of date. Fixed on recent push.

Reply:
LGTM!